test/regress/regress0/printer/learned-lit-output.smt2

```diff
@@ -0,0 +1,9 @@
+; COMMAND-LINE: -o learned-lits
+; SCRUBBER: grep -v -E '(\(learned-lit)'
+; EXPECT: sat
```

Comment:
This would not actually check whether `-o learned-lits` does anything. The other regressions for `-o *` actually check the output...